dashboard/src/features/transactions/actions.js

```diff
@@ -70,16 +70,39 @@ function getTemplateXpubs(tpl) {
 }
 
 form.submitForm = (formParams) => function(dispatch) {
-  const buildPromise = chainClient().transactions.build(builder => {
-    const processed = preprocessTransaction(formParams)
+  try {
+    const buildPromise = chainClient().transactions.build(builder => {
+      const processed = preprocessTransaction(formParams)
```

Comment:
I'm liking the last option best - a throw in the builder function should result in a promise rejection. That covers any future issues really nicely.